### ISSUE TITLE
Autoland came over GS on short final and overcompensated - almost crashed (v2.3 cloned from repo)

### ISSUE THREAD

**Issue by prokopiu** [mSparks43/747-400#807]
Autoland was fine until 200 to 300 m before the runway, it then got above the GS and then overcompensated, taking the nose down and almost crashed into the gras before the runway. I disabled the A/P and somehow managed to land ok :-)
![image](https://user-images.githubusercontent.com/68238857/174403968-db61d4c6-280d-4950-9083-759a2a02ba18.png)

**Comment by mSparks43** [comment]
There's actually warnings about this in the fcom.
but yes, GS needs a little work to make it more robust.

**Comment by prokopiu** [comment]
V2.3 release on my last landing the autoland worked quite well. We were slightly right of the centerline on touch-down. The speed went slightly into the orange range (slightly slow) which caused the autopilot to disconnect mid approach. But as soon as the speed was in nromal range I was able to reactivate the autopilot again. Autoland continued as if nothing has happened. Probably my flaps schedule was not quite right when we entered slightly low speed.

Is it intentional that the A/P disconnects so quickly?
